buyer/receive/receive-order-response-api.md

```diff
@@ -0,0 +1,169 @@
+
+Use `GET` when:
+
+* Same as `POST` and `PUT` above, but:
+* You need to receive the **complete** order with **all** the order lines, regardless they are changed or not.
+
+{% hint style="info" %}
+Pro's:
+
+* Semi real time, receive the order within seconds.
+* You can filter on which order events to receive (in the company integration settings only).
+* You do not have to build or configure the polling pattern.
+
+Con's:
+
+* You need to fetch the order.
+* Basic authentication is not yet supported for the GET `/order/:orderId` API. Send a support request if you need it.
+* You cannot see what order event happened.
+* You need to build or configure a webhook at your side.
+* You need to publish the webhook on the internet (web server and firewall required).
+* You need to obtain and configure a public SSL certificate.
+
+{% endhint %}
+
+## The polling pattern
+Check if there are new or updated orders every polling period, typically 5 minutes, by using the last updated date time stamp of the last fetched order.
+
+The polling pattern is most suitable for companies with low volume order responses and not willing to invest in a web server, firewall and SSL certificate.
+
+{% hint style="info" %}
+Pro's:
+
+* No webhook needed: no web server, firewall or SSL certificate needed.
+
+Con's:
+
+* Not real time, a polling period is typically 5 mins. which is good enough for low volume.
+* You need to build or configure a periodic polling pattern at your side.
+* You cannot filter on which order events to receive, you will receive any order line change.
+* You cannot see what order event happened.
+
+{% endhint %}
+
+### Polling usage
+
+#### Step 1. Fetch updated orders periodically
+
+Fetch every polling period, typically 5 minutes, all orders which are new or changed since last date time.
+
+* Use the latest `lastUpdatedAt` from previous poll request in the `lastUpdatedSince` filter.
+* Sorting is set automatically to `lastUpdatedAt` order `asc` (latest `lastUpdatedAt` will be in the last order in the response body)
+* Set `limit` to the maximum of `100` orders.
+* Optionally use `offset` for paging, but if you receive more than `100` orders, it is easier to reduce the polling period, so you receive less orders per request.
+
+{% api-method method="post" host="https://api.accp.tradecloud1.com/v2" path="/order-search/search" %}
+{% api-method-summary %}
+Search orders
+{% endapi-method-summary %}
+
+{% api-method-description %}
+
+{% endapi-method-description %}
+
+{% api-method-spec %}
+{% api-method-request %}
+{% api-method-headers %}
+{% api-method-parameter name="Authorization" type="string" required=true %}
+Basic Authentication or Bearer Access-Token
+{% endapi-method-parameter %}
+
+{% api-method-parameter name="Content-Type" type="string" required=true %}
+application/json
+{% endapi-method-parameter %}
+{% endapi-method-headers %}
+
+{% api-method-body-parameters %}
+{% api-method-parameter name="body" type="object" required=true %}
+{
```

Comment:
@marcmatt  Do we need to specify here other filters like `companyId`, `contact` etc?
Also json is not properly formatted in the gitbook

![image](https://user-images.githubusercontent.com/53040036/117289956-e74e3100-ae75-11eb-89bd-b0669f3802c8.png)

Reply:
Other fields are not relevant for polling.
I tried to use markdown inside the `api-method-parameter` but that is not allowed
![image](https://user-images.githubusercontent.com/1092388/117306037-a0fdcf80-ae7f-11eb-9fbc-189d114cd09c.png)
I cannot find documentation about this.

Reply:
Ok, let's leave as is then